boat_nav/src/path_planners/planner_base.py

```diff
@@ -126,10 +123,7 @@ def clear_waypoints():
 
 	@staticmethod
 	def update_waypoints(new_pts):
```

Comment:
For this and the other setters, is anything outside of the derived class implementation calling them?  If not should they be private?

Reply:
Believe they used to be, but you're right they're all only used internally now. Fixed.